app/views/tags/show.html.erb

```diff
@@ -24,6 +24,9 @@
   <!--main content-->
   <p><%= ts("This tag belongs to the %{tag_class} Category.", :tag_class => @tag.class::NAME) %>
   <% if @tag.canonical %>
+    <% if @tag.is_a?(Fandom) %>
+      <%= ts("A list of all the %{fandom_relationship_tags_link} is available.", fandom_relationship_tags_link: link_to(ts("Relationship tags in this fandom"), fandom_path(@tag.name))).html_safe %>
```

Comment:
If you could use `t` instead of `ts` here and move the text to the locale files, that would be great! Aside from that, this looks good.